sdk/cosmos/tests/setup.rs

```diff
@@ -16,6 +15,13 @@ fn get_account() -> String {
     std::env::var("COSMOS_ACCOUNT").expect("Set env variable COSMOS_ACCOUNT first!")
 }
 
+fn get_authorization_token() -> Result<AuthorizationToken, AuthorizationTokenParsingError> {
+    let key =
+        std::env::var("COSMOS_MASTER_KEY").expect("Set env variable COSMOS_MASTER_KEY first!");
+
+    AuthorizationToken::primary_from_base64(&key)
```

Comment:
We also don't want to require that the auth key is set when running the tests on replay mode. Similar to how we're first checking for the `TESTING_MODE_KEY` before getting the account name, we should do the same for the authorization token.